quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/cli/PicocliTest.java

```diff
@@ -386,4 +406,18 @@ public void fastStartOptimizedSucceeds() {
         assertEquals(Integer.MAX_VALUE, nonRunningPicocli.exitCode); // "running" state
     }
 
+    @Test
+    public void warnDBRequired() {
+        // dev profile has a default
+        NonRunningPicocli nonRunningPicocli = pseudoLaunch("start-dev");
+        assertEquals(CommandLine.ExitCode.OK, nonRunningPicocli.exitCode);
```

Comment:
Should we also assert the warning is not present in this case?

Reply:
Yes, added now.